### ISSUE TITLE
[discussion] use of open source / free software / FOSS / FLOSS

### ISSUE THREAD

**Issue by japhir** [alan-turing-institute/the-turing-way#2213]
### Summary

In the chapter /reproducible-research/open/open-source.html the chapter is called open software and the terminology is the same. In this issue I'd like to open a discussion about whether "open source" is the right term for what we mean.

### What needs to be done?

Read this discussion on the term "open source" vs. "free software" vs. "free and open source software (FOSS)" vs. "free/libre open source software (FLOSS)"
- Richard Stallman's post on [Why Open Source misses the point](https://www.gnu.org/philosophy/open-source-misses-the-point.en.html)
- A reply that discusses some of the nuances [What's the difference between open source software and free software?](https://opensource.com/article/17/11/open-source-or-free-software)

- [ ] Have a discussion here in the comments about the different terminologies.
- [ ] Decide on the terminology that fits best.
- [ ] Perhaps add some discussion on this to the chapter as well.

### Who can help?

I'm not sure yet :)

---

**Comment by japhir** [comment]
My thoughts are somewhat in agreement with the open source article, where they say many people don't understand the difference between free as in freedom and free/gratis. I therefore have a slight preference for FOSS or FLOSS.

**Comment by tomichec** [comment]
I strongly agree with this point especially in the relation of ethical research. The free/libre software movement focuses on the ethical issues and essential rights of computer users. Open source distract the focus on the technical superiority of creating software collaboratively:

https://www.gnu.org/philosophy/open-source-misses-the-point.en.html
